index.html

```diff
@@ -9,13 +9,25 @@
     <link rel="stylesheet" href="styles.css">
   </head>
   <body>
-  
-    <!-- START HEADER -->
+   <header>
 
-    <!-- END HEADER -->
+    <h1>Juan Fabrega</h1> <h3>posts</h3><h3>about</h3>
```

Comment:
These h3 tags that you have for the navigation links I don't think are really headings. instead let's change this structure a bit. We will keep the `header` element you have, that is good, but inside the header there will be a `<nav>` element. Nav is short for navigation which is basically what this is for this website.

inside the nav we will have a right and left side, as shown in the reference image I sent to you. The left has the title and the right has the list of links. It should look something like this, but feel free to modify it as needed:

```
<header>
  <nav class="nav">
     <div class="nav-left">
       <h1 class="nav-title"></h1>
     </div>
     <div class="nav-right">
       <ul class="nav-link-list">
         <li class="nav-link">
           <a></a>
          </li>
          <li class="nav-link">
            <a></a>
          </li>
       </ul>
     </div>
  </nav>
</header>
```